presto-docs/src/main/sphinx/release/release-348.md

```diff
@@ -0,0 +1,89 @@
+# Release 348 (14 Dec 2020)
+
+## General Changes
+
+* Add support for `DISTINCT` clause in aggregations within correlated subqueries. ({issue}`5904`)
+* Support `SHOW STATS` for arbitrary queries. ({issue}`3109`)
+* Improve query performance by reducing worker to worker communication overhead. ({issue}`6126`)
+* Improve performance of `ORDER BY ... LIMIT` queries. ({issue}`6072`)
+* Reduce memory pressure and improve performance of queries involving joins. ({issue}`6176`)
+* Fix ``EXPLAIN ANALYZE`` for certain queries that contain broadcast join. ({issue}`6115`)
+* Fix planning failures for queries that contain outer joins and aggregations using `FILTER (WHERE <condition>)` syntax. ({issue}`6141`)
+* Fix incorrect results when correlated subquery in join contains aggregation functions such as `array_agg` or `checksum`. ({issue}`6145`)
+* Fix incorrect query results when using `timestamp with time zone` constants with precision higher than 3 
+  describing same point in time but in different zones. ({issue}`6318`)
+* Fix duplicate query completion events if query fails early. ({issue}`6103`)
+* Fix query failure when views are accessed and current session does not
+  specify default schema and catalog. ({issue}`6294`)
+
+## Web UI
+
+* Add support for OAuth2 authorization. ({issue}`5355`)
+* Fix invalid operator stats in Stage Performance view. ({issue}`6114`)
+
+## JDBC Driver Changes
+
+* Allow reading `timestamp with time zone` value as `ZonedDateTime` using `ResultSet.getObject(int column, Class<?> type)` method. ({issue}`307`)
+* Accept `java.time.LocalDate` in `PreparedStatement.setObject(int, Object)`. ({issue}`6301`)
+* Extend `PreparedStatement.setObject(int, Object, int)` to allow setting `time` and `timestamp` values with precision higher than nanoseconds. 
```

Comment:
This description is long and still does not pass all the information needed to do the actual call (what value of `java.sql.Types` should be used).
I think we should either make it shorter; like:
```
Extend `PreparedStatement.setObject(int, Object, int)` to allow setting `time` and `timestamp` values with precision higher than nanoseconds. 
This can be done via `String` object representing valid SQL literal.
```
Or extend the `alongside...` part.

cc: @findepi